.gitignore

```diff
@@ -19,4 +19,4 @@ dist
 
 # Ponder
 generated
-ponder
```

Comment:
This change allows to narrow down the ignore path and allow creating `ponder` directories in the monorepo file tree.